src/routes/report-route.tsx

```diff
@@ -55,6 +67,14 @@ function createReportRoute(
     const env = envParam || DEFAULT_ENV;
     const funnelMode = funnelModeParam || DEFAULT_FUNNEL_MODE;
     const scale = scaleParam || DEFAULT_SCALE;
+    const extra = extraParam === "true";
+    const breakout = breakoutParam ? breakoutParam === "on" : extra;
+
+    const reportControls = controls || [];
```

Comment:
What's this used for?

Reply:
whoops, I forgot to pass it in to the `ReportFilterControls` 🤦

Reply:
fixed 2324d37c88ddd22eedbe838f5988e31ef604d14c